doc/developer_guide/developer_guide.md

```diff
@@ -0,0 +1,130 @@
+# Developer Guide
+
+This document contains developer information on how to build, run, modify and publish this Lua project.
+
+## Prerequisites
+
+This project needs a Lua interpreter &ge; Lua 5.1.
+
+## Lua Rocks 3
+
+To install the dependencies you also need the [LuaRocks](https://luarocks.org/) package manager. While LuaRocks version 3 is already available for quite a while, it has not yet reached all package repostories.
+
```

Comment:
You can install the dependencies then with
```
luarocks install --deps-only *.rockspec
```

Reply:
Done.